<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReactorControlRods</class>
 <widget class="QDialog" name="ReactorControlRods">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reactor Control Rods</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Control Rods</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="buttonHold">
        <property name="text">
         <string>Hold</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="speedLayout">
        <item>
         <widget class="QPushButton" name="buttonSlow">
          <property name="text">
           <string>S</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonMedium">
          <property name="text">
           <string>M</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonFast">
          <property name="text">
           <string>F</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="buttonInsert">
        <property name="text">
         <string>Insert</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="buttonPull">
        <property name="text">
         <string>Pull</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="buttonCenter">
        <property name="text">
         <string>Center only</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupRods">
     <property name="title">
      <string>Control Rods</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
